introduction.md

```diff
@@ -1,2 +1,4 @@
 Hello, my name is ....
 The fav codeschool path is: Ruby. Why? Because there is no Emerald path.
+* Linux
+* GitHub WIP
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?